dotcom-rendering/src/web/components/MatchTabs.tsx

```diff
@@ -0,0 +1,90 @@
+import { css } from '@emotion/react';
+import { space } from '@guardian/src-foundations';
+import { headline } from '@guardian/src-foundations/typography';
+import { decidePalette } from '@root/src/web/lib/decidePalette';
+
+type Props = {
+	minByMinUrl?: string;
+	reportUrl?: string;
+	format: ArticleFormat;
+};
+
+const thinGreySolid = (palette: Palette) =>
+	`1px solid ${palette.border.secondary}`;
+
+const GreyBorder = ({ palette }: { palette: Palette }) => (
+	<div
+		css={css`
+			/* stylelint-disable-next-line color-no-hex */
+			border-left: ${thinGreySolid(palette)};
+			margin-left: ${space[1]}px;
+			width: ${space[2]}px;
+		`}
+	/>
+);
+
+const tabsContainer = (palette: Palette) => css`
+	display: flex;
+	position: relative;
+	border-bottom: ${thinGreySolid(palette)};
+`;
+
+const tab = (palette: Palette) => css`
+	flex-basis: 50%;
+	height: 40px;
+	border-top: 3px solid ${palette.border.secondary};
+
+	:nth-child(1) {
```

Comment:
It's best to avoid these selectors in css. In fact, it's best to avoid _any_ selectors.

If there's a need to add css to a child component then that means we probably want to create a new css class.

Note. the thing with nth-child selectors is they tend to cause issues with some libraries. I _think_ Emotion 11 has some issues with them when extracting critical css?

Reply:
Just as a note, these were there before I started work on MatchTabs.